cpp/core/CMakeLists.txt

```diff
@@ -328,3 +328,8 @@ install(TARGETS gluten
         DESTINATION ${CMAKE_INSTALL_LIBDIR})
 install(FILES ${CMAKE_CURRENT_SOURCE_DIR}/resources/libhdfs.so
         DESTINATION ${LIBHDFS3_DESTINATION})
+
+add_custom_command(TARGET gluten POST_BUILD COMMAND ld $<TARGET_FILE:gluten> || true
+        COMMENT "Checking ld result of libgluten.so")
+add_custom_command(TARGET gluten POST_BUILD COMMAND ldd $<TARGET_FILE:gluten> || true
+        COMMENT "Checking ldd result of libgluten.so")
```

Comment:
Adding short CMake messages to check ld / ldd result of the built library. Could be useful to help troubleshooting symbol-related issues in CI.

Reply:
any particular reason to keep these commands in git repo? build/link issue should be fixed before changes merged, keep above commands in git bring confused/noisy information when first build and every after.

if it's really useful, maybe we could move it to developer docs.